server/controller/rides.js

```diff
@@ -3,69 +3,56 @@ import client from '../db/index';
 class RidesController{
     static getAllRides(req, res, next){
         client.query("select * from rides", (err, data) => {
-            if (err) return next(err);
+            if (err) return next(err.message);
```

Comment:
why do we still have next here and not returning the error to the user the way you have been returning it?